app/email/notify-send.js

```diff
@@ -84,7 +84,7 @@ const sendFarmerApplicationEmail = async (data, blob) => {
   let emailTemplateId = templateIdFarmerApplicationGeneration
   let isSuccess = true
   if (endemics.enabled) {
-    emailTemplateId = emailTemplateIdSelector(data.userType, data.oldWorldRejectedAgreement10months?.isExistingUserRejectedAgreementWithin10months)
+    emailTemplateId = emailTemplateIdSelector(data.userType, data.userTypeStatus?.isExistingUserRejectedAgreementWithin10months)
```

Comment:
What does the emailTemplateIdSelector function do? Do we need to just send it the userTypeStatus rather than specifically the 'isExistingUserRejectedAgreementWithin10months' ?

Reply:
we need to because the text should change for the other variable but we decided to stay with the old template. In the future if the text changes this will be needed to check before selecting template using the Id